net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils autotools
+
+DESCRIPTION="Facebook protocol plugin for BitlBee"
+HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug"
+
+if [ "${PV}" = "9999" ] ; then
+	inherit	git-r3
+	EGIT_REPO_URI="https://github.com/bitlbee/bitlbee-facebook"
+else
+	SRC_URI="https://github.com/bitlbee/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/json-glib
+	>=net-im/bitlbee-3[plugins]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable debug) \
+		--enable-minimal-flags
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```

Comment:
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

	find "${D}" -name '*.la' -delete || die

and then remove inheriting `eutils`

Reply:
Ok, thanks.